dotcom-rendering/src/web/components/ElementContainer.tsx

```diff
@@ -81,7 +83,7 @@ export const ElementContainer = ({
 			// Create a react element from the tagName passed in OR
 			// default to <div>
 			return _jsx(`${element}`, {
-				className: style,
+				className: className ? `${style} ${className}` : style,
```

Comment:
Alternatively, if we have a default classname set, we shouldn't need to check for its existence

Reply:
I wondered, but ElementContainer is quite generic - we return a component with a variable tag name, so I didn't know whether a default class name would always make sense... what do you think?